process_lunr.xml

```diff
@@ -128,7 +147,12 @@
   <!--
     Check if Node.js is installed.
   -->
-  <target name="lunr.node.check">
+  <target
+    name="lunr.node.check"
+    if="is.Bootstrap"
```

Comment:
ibid

Reply:
Dependency of `lunr` target, it is only fired if `is.Bootstrap` has been set within the `lunr.init` target, which means `bootstrap.process.pre` has been fired.